index.js

```diff
@@ -130,8 +130,15 @@ class CreateCertificatePlugin {
       .then(() => this.serverless.cli.log('serverless-certificate-creator: disabled.'));
   }
 
-  listCertificates() {
-    return this.acm.listCertificates({}).promise();
+  listCertificates(NextToken) {
```

Comment:
nit: would be good to use `async/await` instead of `.then`, to ensure a Promise is always returned and improve readability.

Reply:
I'm not sure about this one, because I would expect, that all of the Serverless plugins code should match ES5, because there is no transpilation done. Are you sure, that we are good to use async/await? I thought that's one of the reasons, why this plugin uses `.then`

Reply:
Good point - I thought we were already using `async/await` somewhere else but I realised I was thinking of `serverless` itself 😅